test/test_listen_callback.cpp

```diff
@@ -22,26 +195,12 @@ srt_listen_callback_fn SrtTestListenCallback;
  *  - rejected connection with wrong password
  *  - rejected connection on nonexistent user
 */
-TEST(Core, ListenCallback) {
 
-    using namespace std;
-
-    ASSERT_EQ(srt_startup(), 0);
-
-    // Create server on 127.0.0.1:5555
-
-    const SRTSOCKET server_sock = srt_create_socket();
-    ASSERT_GT(server_sock, 0);    // socket_id should be > 0
+TEST(Core, DISABLED_ListenCallback)
```

Comment:
To remove? As I read it, the test is replaced by those three tests based on the fixture.

Reply:
Ops. Forgot to remove it, didn't want to do this until I'm certain that everything is working. Just a moment ;)